src/pass.cpp

```
@@ -31,6 +32,18 @@ using Enums::PASS_OTP_GENERATE;
 using Enums::PASS_REMOVE;
 using Enums::PASS_SHOW;
 
+namespace {
+QString effectiveCharset(const PasswordConfiguration &passConfig) {
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)